projects/angular-split/src/lib/split/split.component.ts

```diff
@@ -164,24 +165,18 @@ export class SplitComponent {
   constructor() {
     if (isDevMode()) {
       // Logs warnings to console when the provided areas sizes are invalid
-      effect(() => {
-        // Special mode when no size input was declared which is a valid mode
-        if (this.unit() === 'percent' && this._visibleAreas().every((area) => area.size() === 'auto')) {
-          return
-        }
+      afterRenderEffect({
```

Comment:
I think we need a note comment here. I think effect is more correct here than afterRenderEffect but we are forced to use it until angular fixes https://github.com/angular/angular/issues/59067.
Or do you think this is the preferred solution even after the issue is fixed?

Reply:
I think it is indeed the preferred solution. 
Let me explain. The `afterRender({read: () => ...})` runs when all Angular processing is done.  The code we run is _not_ going to change anything Angular should care about, so it is the correct abstraction